pkg/detector/ospkg/amazon/amazon.go

```diff
@@ -23,6 +23,7 @@ var (
 		"2": time.Date(2024, 6, 30, 23, 59, 59, 0, time.UTC),
 		// N/A
 		"2022": time.Date(3000, 1, 1, 23, 59, 59, 0, time.UTC),
+		"2023": time.Date(3000, 1, 1, 23, 59, 59, 0, time.UTC),
```

Comment:
>Starting with AL2023, a new Amazon Linux major release will be available every 2 years. This cadence provides you with a more predictable release cycle and up to 5 years of support, making it easier for you to plan your upgrades.

According to this statement, we can put 15 Mar 2028 as the EOL date. We can update it after Amazon announces the exact date.

https://endoflife.date/amazon-linux